Update attestation-container document (#5109)
microsoft/CCF: attestation-container/README.md

## How to start the app

> The following command starts the gRPC server application inside SEV-SNP VM.

Reviewer: ```suggestion
The following command starts the gRPC server application (must be inside SEV-SNP VM).
```

To make it clear that `go run .` doesn't start a VM.

```bash
# In the same directory as this README.md
go run .
```

> You can use insecure virtual mode to run the application without SEV-SNP VM

Reviewer: ```suggestion
You can use insecure virtual mode to run the application on non SEV-SNP VM.
```

(**It't not secure. Do not use it in production**).

```bash
go run . --insecure-virtual
```

> You can find the details of the flag and other flags in [attestation-container.go](https://github.com/microsoft/CCF/blob/main/attestation-container/attestation-container.go)

Reviewer: Simply run `go run . --help` no?

## Build

> Since it's a go application, you can build the application before run it.

Reviewer: ```suggestion
Since it's a go application, you can build the application before running it.
```
